dev/io.openliberty.org.jboss.weld4/bnd.overrides

```diff
@@ -8,16 +8,16 @@ Build-Identifier: SNAPSHOT-${now}
",org.jboss.weld.bootstrap;version="4.0.1";uses:="jakarta.enterprise.context.spi,jakarta.enterprise.inject.spi,org.jboss.weld.bean,org.jboss.weld.bean.builtin,org.jboss.weld.bootstrap.api,org.jboss.weld.bootstrap.api.helpers,org.jboss.weld.bootstrap.spi,org.jboss.weld.event,org.jboss.weld.exceptions,org.jboss.weld.logging,org.jboss.weld.manager,org.jboss.weld.manager.api,org.jboss.weld.module,org.jboss.weld.resources.spi",org.jboss.weld.config;version="4.0.1";uses:="org.jboss.weld.bootstrap.api,org.jboss.weld.bootstrap.spi",org.jboss.weld.event;version="4.0.1";uses:="jakarta.enterprise.context.spi,jakarta.enterprise.event,jakarta.enterprise.inject.spi,jakarta.enterprise.util,org.jboss.weld.bean,org.jboss.weld.bean.builtin,org.jboss.weld.bootstrap.api,org.jboss.weld.events,org.jboss.weld.injection,org.jboss.weld.manager,org.jboss.weld.module,org.jboss.weld.util",org.jboss.weld.exceptions;version="4.0.1";uses:="jakarta.enterprise.inject,jakarta.enterprise.inject.spi",org.jboss.weld.injection;version="4.0.1";uses:="jakarta.enterprise.context.spi,jakarta.enterprise.inject.spi,jakarta.enterprise.inject.spi.configurator,org.jboss.weld.bootstrap.api,org.jboss.weld.injection.spi,org.jboss.weld.manager",org.jboss.weld.interceptor.proxy;version="4.0.1";uses:="jakarta.enterprise.context.spi,jakarta.enterprise.inject.spi,jakarta.interceptor,org.jboss.weld.bean.proxy,org.jboss.weld.interceptor,org.jboss.weld.interceptor.spi.metadata,org.jboss.weld.interceptor.spi.model,org.jboss.weld.manager,org.jboss.weld.util",org.jboss.weld.interceptor.util.proxy;version="4.0.1";uses:="org.jboss.weld.bean.proxy",org.jboss.weld.logging;version="4.0.1";uses:="jakarta.enterprise.context.spi,jakarta.enterprise.inject.spi,javax.naming,org.jboss.logging,org.jboss.weld.exceptions",org.jboss.weld.manager;version="4.0.1";uses:="jakarta.el,jakarta.enterprise.context.spi,jakarta.enterprise.event,jakarta.enterprise.inject,jakarta.enterprise.inject.spi,jakarta.enterprise.inject.spi.configurator,org.jboss.weld.bean,org.jboss.weld.bean.proxy,org.jboss.weld.bootstrap.api,org.jboss.weld.bootstrap.spi,org.jboss.weld.construction.api,org.jboss.weld.ejb.spi,org.jboss.weld.event,org.jboss.weld.events,org.jboss.weld.inject,org.jboss.weld.interceptor.spi.model,org.jboss.weld.manager.api,org.jboss.weld.serialization.spi",org.jboss.weld.security;version="4.0.1";uses:="org.jboss.weld.security.spi",org.jboss.weld.util;version="4.0.1";uses:="jakarta.el,jakarta.enterprise.context.spi,jakarta.enterprise.event,jakarta.enterprise.inject,jakarta.enterprise.inject.spi,jakarta.enterprise.util,jakarta.interceptor,org.jboss.weld.bean,org.jboss.weld.bootstrap,org.jboss.weld.bootstrap.api,org.jboss.weld.bootstrap.spi,org.jboss.weld.exceptions,org.jboss.weld.inject,org.jboss.weld.injection,org.jboss.weld.interceptor.spi.model,org.jboss.weld.manager,org.jboss.weld.resources.spi,org.jboss.weld.serialization.spi",org.jboss.weld.xml;version="4.0.1";uses:="org.jboss.weld.bootstrap.spi,org.xml.sax,org.xml.sax.helpers"
```

Comment:
was this section kept the same as before or re-copied from the 4.0.1 weld osgi bundle?

Reply:
Github is just highlighting the whole thing even though the only change is a lot of 4.0.0 into 4.0.1